lib/iron_bank/csv.rb

```diff
@@ -4,23 +4,30 @@ module IronBank
   # A custom CSV converter
   #
   class CSV < ::CSV
+    DECIMAL_INTEGER_REGEX = /^[+-]?\d+$/.freeze
+    DECIMAL_FLOAT_REGEX   = /^[+-]?(?:\d*\.|\.\d*)\d*$/.freeze
+
     CSV::Converters[:decimal_integer] = lambda { |field|
       begin
+        return field unless field
+
         encoding = field.encode(CSV::ConverterEncoding)
 
         # Match: [1, 10, 100], No match: [0.1, .1, 1., 0b10]
-        /^[+-]?\d+$/.match?(encoding) ? encoding.to_i : field
+        DECIMAL_INTEGER_REGEX.match?(encoding) ? encoding.to_i : field
       rescue # encoding or integer conversion
```

Comment:
What do you think of removing entirely the `rescue` block to let the exception bubble up since we deemed that the performances of going through a succession of `raise/rescue` are not acceptable?

Reply:
👍 I would be hesitant as a general purpose converter, but as these are used for a very specific use-case (Zuora CSVs) I think it's ok to make the assumption that the encoding & regex matching will work correctly.